assemble/bin/accumulo-cluster

```diff
@@ -32,9 +32,9 @@ Commands:
   stop                       Stops Accumulo cluster
   kill                       Kills Accumulo cluster
   start-non-tservers         Deprecated. Starts all services except tservers
```

Comment:
I'm wondering if we should remove the deprecated commands at this point. Were they marked deprecated in an older release?

Reply:
Yes they were marked deprecated in 2.1. I was going to put up another PR removing them.

Reply:
Ended up just making the changes here to avoid merge conflicts.